README.md

```diff
@@ -294,7 +308,8 @@ Workflow:
     1.  determine containerPort list
 1.  Determine if the pod needs to be setup for Istio sidecar proxy
     1.  if pod has a container named `istio-proxy` AND pod has more than 1 container
-        1.  Final Logic TBD -- e.g. pod labels?  namespace checks?
+        1.  if pod has annotation with key `sidecar.istio.io/inject` with value `false` then skip redirect
+        1.  else do redirect
```

Comment:
nit: capitalize first character in both points.

Reply:
fixed other points around this one